web/src/beta/features/Editor/SketchLayerManager/index.tsx

```diff
@@ -55,20 +57,26 @@ const SketchLayerManager: React.FC<SketchProps> = ({ sceneId, layerStyles, onClo
   }, []);
 
   const handleSubmit = () => {
+    const schemaJSON = propertyList.reduce((acc, property, index) => {
+      const [key] = Object.keys(property);
+      const value = `${property[key]}_${index + 1}`;
```

Comment:
Why do we need the value to be like `Int_1`, is this requirement from backend?

Reply:
Oh it's for the sort? Is it necessary?

Reply:
Emmm maybe it's okey, but better add a comment like `// record order for sort later`, i feel it is a little bit difficult for other developers to understand on first read.

Reply:
@airslice yes I will add the comment, this is for sorting cause when the custom properties send to backend they are ordered in alphabetic order, but what we want is user to see the custom properties in the order  he added them